nixos/modules/services/mail/postfix.nix

```diff
@@ -673,6 +673,7 @@ in
       services.mail.sendmailSetuidWrapper = mkIf config.services.postfix.setSendmail {
         program = "sendmail";
         source = "${pkgs.postfix}/bin/sendmail";
+        owner = "nobody";
```

Comment:
Same thing here with all of these.

Reply:
cc @globin @dotlambda @lewo postfix maintainers: what owner should /run/wrappers/bin/sendmail have? As well as the other wrappers below